pkgs/by-name/nv/nvidia-container-toolkit/package.nix

```diff
@@ -104,7 +103,7 @@ buildGoModule rec {
         --prefix PATH : ${libnvidia-container}/bin
 
       mkdir -p $tools/bin
-      mv $out/bin/{containerd,crio,docker,nvidia-toolkit,toolkit} $tools/bin
+      mv $out/bin/{nvidia-cdi-hook,nvidia-container-runtime.cdi,nvidia-container-runtime-hook,nvidia-container-runtime.legacy} $tools/bin
```

Comment:
Nit for the future: what if we override `src` with a revision where some of these do not exist?

Reply:
Good point, will update the derivation.